server/config/config.go

```diff
@@ -34,12 +35,16 @@ import (
 
 // ServerConfig holds the configuration of etcd as taken from the command line or discovery.
 type ServerConfig struct {
-	Name           string
-	DiscoveryURL   string
-	DiscoveryProxy string
-	ClientURLs     types.URLs
-	PeerURLs       types.URLs
-	DataDir        string
+	Name              string
+	EnableV2Discovery bool
```

Comment:
nit: would be nice to group all discovery configuration together. I don' that name is part of disovery.

Reply:
Thanks & Fixed.